homeassistant/components/camera/__init__.py

```diff
@@ -906,7 +908,10 @@ def camera_capabilities(self) -> CameraCapabilities:
                 if self._webrtc_provider or self._legacy_webrtc_provider:
                     frontend_stream_types.add(StreamType.WEB_RTC)
 
-        return CameraCapabilities(frontend_stream_types)
+        return CameraCapabilities(
```

Comment:
Is the CameraEntityFeature not available in the front end?
If yes, we don't need to extend the CameraCapabilities

Reply:
It's not currently in the front end but it could be added. What's the guidance on the when to use the `CameraEntityFeature` vs the `CameraCapabilities`? It seems like the `CameraCapabilities` is created from the `CameraEntityFeature`.